### PR TITLE
add new story feature

### PR DESCRIPTION
**Pull request by patrickwregan**

### CODE REVIEW

**Comment by patrickwregan** [comment]
please review this @casey-collab thanks!

File: english/wonder-about/wonder.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+You conclude that marshmallow muscles don't suffice to steal your lighter so
```
Comment (casey-collab) [inline english/wonder-about/wonder.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by patrickwregan** [comment]
please try again @casey-collab , thank you.

File: english/wonder-about/wonder.md (hunk 1 above)
Comment (casey-collab) [inline english/wonder-about/wonder.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Looks good to me! :zap:

**Comment by cbuckey-uda** [comment]
Oh, sorry, I can't merge this because there are merge conflicts. Could you merge the master branch of the original repository into your branch and then update the pull request?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to resolve the merge conflicts, feel free to re-open it.

**Pull request closed without merging**